models/reddit_ads.yml

```diff
@@ -154,12 +154,12 @@ models:
             - post_id
             - currency
     columns:
-      - name: date_day
-        description: '{{ doc("date_day") }}'
+      - name: date_day 
+        description: '{{ doc("date_day") }}' 
```

Comment:
Should we apply not_null tests to these fields?

Reply:
updated